kernel/packages/unity-interface/UnityInterface.ts

```diff
@@ -306,6 +306,10 @@ export class UnityInterface {
     this.gameInstance.SendMessage('TutorialController', 'SetTutorialEnabled', JSON.stringify(fromDeepLink))
   }
 
+  public SetTutorialEnabledForUsersThatAlreadyDidTheTutorial() {
+    this.gameInstance.SendMessage('TutorialController', 'SetTutorialEnabledForUsersThatAlreadyDidTheTutorial')
```

Comment:
we really need a better way for this in the future. Shall we create a ticket so we dont forget?

Reply:
I was about to write the same comment 😂
I don't like this approach very much, I have an idea to doing it more scalable but we are now in a hurry, so we can revisit it later

Reply:
Yes, I agree. In this case we have needed to apply these changes on the tutorial flow a day before the release and it has been the quicker way. I will annote this for a future improvement.